types.go

```diff
@@ -538,3 +553,20 @@ type JobEvent struct {
 	EndLine   int         `json:"end_line"`
 	Verbosity int         `json:"verbosity"`
 }
+
+// User represents an user
+type User struct {
```

Comment:
In my local test, I got user response from awx rest api `/api/v2/users/1` below:

```
{
    "id": 1,
    "type": "user",
    "url": "/api/v2/users/1/",
    "related": {
        "named_url": "/api/v2/users/admin/",
        "admin_of_organizations": "/api/v2/users/1/admin_of_organizations/",
        "authorized_tokens": "/api/v2/users/1/authorized_tokens/",
        "roles": "/api/v2/users/1/roles/",
        "organizations": "/api/v2/users/1/organizations/",
        "access_list": "/api/v2/users/1/access_list/",
        "teams": "/api/v2/users/1/teams/",
        "tokens": "/api/v2/users/1/tokens/",
        "personal_tokens": "/api/v2/users/1/personal_tokens/",
        "credentials": "/api/v2/users/1/credentials/",
        "activity_stream": "/api/v2/users/1/activity_stream/",
        "projects": "/api/v2/users/1/projects/"
    },
    "summary_fields": {
        "user_capabilities": {
            "edit": true,
            "delete": true
        }
    },
    "created": "2018-05-22T06:08:05.932739Z",
    "username": "admin",
    "first_name": "Kai",
    "last_name": "Ying",
    "email": "admin@kingnet.com",
    "is_superuser": true,
    "is_system_auditor": false,
    "ldap_dn": "",
    "external_account": null,
    "auth": []
}
```

Can you confirm the struct fields and ensure there is no fields left?

Reply:
I just checked. Some elements in the "Related" structure are missing. Will fix it now.

Reply:
Not ONLY `related` fields, but also some struct fields, like `User`, `Ldap` field is not exists in ListUsers response, and some fields like `is_system_auditor `, `ldap_dn`, `external_account` etc

Feel free to add comments!

Reply:
Following the doc https://docs.ansible.com/ansible-tower/2.3.0/html/towerapi/users.html
is_system_auditor and external_account does not exist.

Reply:
Check the new version document: https://docs.ansible.com/ansible-tower/3.2.4/html/towerapi/users.html for details.

Reply:
Ah alright. my bad. Will fix it now.